app/views/unattended/provisioning_templates/snippet/preseed_kernel_options_autoinstall.erb

```diff
@@ -30,7 +36,18 @@ description: options for the kernel / preseed startup initialization
     options << "ip=[#{iface.ip6}]::[#{subnet6.gateway}]:[#{subnet6.mask}]:#{hostname}:#{iface.identifier}:none:[#{subnet6.dns_servers.join(']:[')}]"
   end
 
+  options << 'ramdisk_size=1500000'
+  options << 'fsck.mode=skip'
+  options << 'autoinstall'
+  options << "url=http://#{@preseed_server}#{image_path}"
+  options << 'cloud-config-url=/dev/null'
+  if @add_userdata_quotes
+    options << "\"#{userdata_option}\""
+  else
+    options << userdata_option
+  end
```

Comment:
One slight change from what you suggested is pulling out the duplicated building of this option into a variable.

Reply:
Looks good to me :+1: